extensions/rke1/nodetemplates/vsphere/create.go

```diff
@@ -14,6 +14,7 @@ const vmwarevsphereNodeTemplateNameBase = "vmwarevsphereNodeConfig"
 // an VSphere node template and returns the NodeTemplate response
```

Comment:
There's a diff, here cause of `MergeOverride`.  It seems good to me, should be the cause of 2.7 vs 2.8 & 2.9